README.md

```diff
@@ -1,10 +1,73 @@
-## Installation
-Open your terminal then paste the following command to clone this repository.  Upon cloning, you may now import and simulate our source code into **Android Studio**.
-```bash
-git clone git@github.com:DIT112-V21/group-07.git
-```
 
-## What
+<p align="center"><img src="https://i.postimg.cc/tRKJ5w9C/Pathfinder.jpg" alt="Pathfinder.jpg" width="480" height="121"></p>
+<p align="center">Project for autonomous public transportation</p>
+
+[![Arduino CI](https://github.com/DIT112-V21/group-07/actions/workflows/Arduino-Build&Test.yml/badge.svg?branch=master&event=push)](https://github.com/DIT112-V21/group-07/actions/workflows/Arduino-Build&Test.yml)
+[![Arduino CI](https://github.com/DIT112-V21/group-07/actions/workflows/Arduino-Build&Test.yml/badge.svg?branch=master&event=pull_request)](https://github.com/DIT112-V21/group-07/actions/workflows/Arduino-Build&Test.yml)
+
+***
+
+### Contents
+- [Introduction](#introduction)
+- [App Installation](#app-installation)
+- [Dependencies](#dependencies)
+- [Testing](#testing)
+- [What](#what?)
+- [Why](#why?)
+- [How](#howt?)
```

Comment:
this link might not work , should it be (#how?)

Reply:
It should be fixed now.